### ISSUE TITLE
Improve thumbnail images for start page of slideshow

### ISSUE THREAD

**Issue by ggeisler** [projectblacklight/blacklight-gallery#50]
The current approach used for the index page of the slideshow view seems less than ideal in that the thumbnail image for each item is a fairly zoomed-in portion of the larger image. This often results in an thumbnail image that is pretty useless for understanding what the image is really about. A couple of examples:

<img width="609" alt="cape_of_good_hope___maps_of_africa__an_online_exhibit_-_spotlight_at_stanford" src="https://user-images.githubusercontent.com/101482/36220712-e13205e6-1178-11e8-9017-7d8656e73243.png">

---

<img width="322" alt="31411720-5b3d505e-adc7-11e7-81f5-42855a919362_png" src="https://user-images.githubusercontent.com/101482/36276224-765fb59c-124a-11e8-9e44-a752828ba46d.png">

---

I wonder if we can consider using `object-fit: cover;` for these thumbnail images? We use that approach in a couple of places in Spotlight, such as the Administration > Curation > [Items table](https://github.com/projectblacklight/spotlight/blob/b4608610ed0405f1d8ed4d6c9c17a02b06b9bcf6/app/assets/stylesheets/spotlight/_catalog.scss#L14) that shows a thumbnail of each item. As an example, using `object-fit: cover;` the first example above would look like this:

<img width="610" alt="cape_of_good_hope___maps_of_africa__an_online_exhibit_-_spotlight_at_stanford 2" src="https://user-images.githubusercontent.com/101482/36221349-0f079e48-117b-11e8-9dd0-423ba61734f0.png">

And the second example would like this:

<img width="306" alt="thumbnail__yes_-_digital_library_of_the_middle_east_-_dlme_search_results" src="https://user-images.githubusercontent.com/101482/36221432-535982d2-117b-11e8-851c-08b05d11e7b9.png">

These are more visually appealing and provide more info about the item to the user than the current approach.

I think at least some versions of IE don't support `object-fit` so that is one thing to consider, but all other browsers do and I think there are polyfills or other fallback approaches we could use for IE.

**Comment by mejackreed** [comment]
@ggeisler how are you applying the `object-fit: cover` style here? In my tests I haven't been able to reproduce the same results in your screenshot. Is this just on the `img` elements?

**Comment by ggeisler** [comment]
@mejackreed Yes `object-fit` is applied to the `img` but we need to make some other CSS adjustments, too.

Not sure these are the exact selectors currently used but hopefully this makes it clear what elements we want to target:

``` CSS
.slideshow-documents .grid .document .thumbnail {
  padding: 0px;  # add this
}

.slideshow-documents .grid .document .thumbnail a>img {
  height: 100px; # add this
  object-fit: cover; # add this
  width: 100px; # add this
  left: -50%; # remove this
  top: -50%; # remove this
}
```

I just made those changes in the web inspector for both the Spotlight and DLME examples used in my previous screenshots and got the same result as in those screenshots. But without trying to make the changes directly in the blacklight-gallery code and then testing with Spotlight I'm not 100% sure there isn't another CSS adjustment or two that needs to be made in the Spotlight CSS, for example.

**Comment by mejackreed** [comment]
Gotcha thanks @ggeisler . I also just want to reconfirm (with all of these css changes) that we still want to use the Bootstrap `thumbnail` class here
